src/backend/BinaryOperator.js

```diff
@@ -9,16 +9,32 @@ const Node = require('./Node.js');
 
 class BinaryOperator extends Node {
 
-  constructor() {
+  constructor(operatorText) {
```

Comment:
It would be nice to have a constructor that takes in both the operatorText and operands as well. You can do this in JavaScript by using default parameter values (like in python if you are familiar with that)?

Reply:
Added operand parameters as requested